docs/windows.md

```diff
@@ -125,127 +158,32 @@ document.
-    ((Get-Content -path $mountPath/var/lib/kube-proxy/kubeconfig.conf -Raw) -replace '/var',"$($mountPath)/var") | Set-Content -Path /var/lib/kube-proxy/kubeconfig.conf
-    ((Get-Content -path /var/lib/kube-proxy/kubeconfig.conf -Raw) -replace '\/',"/") | Set-Content -Path /var/lib/kube-proxy/kubeconfig.conf
-    sed -i 's/mode: iptables/mode: \"\"/g' $mountPath/var/lib/kube-proxy/config.conf
-    & "$mountPath/k/kube-proxy/kube-proxy.exe" --config=$mountPath/var/lib/kube-proxy/config.conf --v=10 --proxy-mode=userspace --hostname-override=$env:NODE_NAME
-kind: ConfigMap
-metadata:
-  labels:
-    app: kube-proxy
-  name: kube-proxy-windows
-  namespace: kube-system
-```
-
-Set the `hostNetwork` option as true in spec of kube-proxy-windows daemonset.
-
-```yaml
-apiVersion: apps/v1
-kind: DaemonSet
-metadata:
-  labels:
-    k8s-app: kube-proxy
-  name: kube-proxy-windows
-  namespace: kube-system
-spec:
-  selector:
-    matchLabels:
-      k8s-app: kube-proxy-windows
-  template:
-    metadata:
-      labels:
-        k8s-app: kube-proxy-windows
-    spec:
-      hostNetwork: true
-```
+#### Add Windows antrea-agent DaemonSet
 
-For containerd runtime, Set `hostNetwork` as true and add options in spec and command.
+Starting from Antrea 1.13, you need to manually set the `kubeAPIServerOverride` field in the YAML configuration file as the Antrea Proxy `proxyAll` mode is enabled by default.
 
 ```yaml
-apiVersion: apps/v1
-kind: DaemonSet
-metadata:
-  labels:
-    k8s-app: kube-proxy
-  name: kube-proxy-windows
-  namespace: kube-system
-spec:
-  selector:
-    matchLabels:
-      k8s-app: kube-proxy-windows
-  template:
-    metadata:
-      labels:
-        k8s-app: kube-proxy-windows
-    spec:
-      securityContext:
-        windowsOptions:
-          hostProcess: true
-          runAsUserName: "NT AUTHORITY\\SYSTEM"
-      hostNetwork: true
-      serviceAccountName: kube-proxy
-      containers:
-      - command:
-        - pwsh
-        args:
-        - -file
-        - $env:CONTAINER_SANDBOX_MOUNT_POINT/var/lib/kube-proxy-windows/run-script.ps1
+    # Provide the address of Kubernetes apiserver, to override any value provided in kubeconfig or InClusterConfig.
+    # Defaults to "". It must be a host string, a host:port pair, or a URL to the base of the apiserver.
+    kubeAPIServerOverride: "10.10.1.1:6443"
+
+    # Option antreaProxy contains AntreaProxy related configuration options.
+    antreaProxy:
+      # ProxyAll tells antrea-agent to proxy ClusterIP Service traffic, regardless of where they come from.
+      # Therefore, running kube-proxy is no longer required. This requires the AntreaProxy feature to be enabled.
+      # Note that this option is experimental. If kube-proxy is removed, option kubeAPIServerOverride must be used to access
+      # apiserver directly.
+      proxyAll: true
 ```
 
-Then apply the `kube-proxy.yml`.
-
 ```bash
-kubectl apply -f kube-proxy.yml
+# Example:
```

Comment:
I feel it would be better to remove the "Example:" comment from the command, but add a sentence before the command, like: "An example command that downloads the antrea-agent manifest, sets `kubeAPIServerOverride`, and deploys the antrea-agent DaemonSet is like below:".

So, readers can directly copy the command by clicking the copy button on the web page.

Could you also change other commands with the "Example:" comment?

Reply:
Sure, removed all `example` comment and updated sentences.